gammapy/scripts/image_pipe.py

```diff
@@ -64,9 +66,34 @@ def make_counts(self, obs_id):
         obs = self.data_store.obs(obs_id=obs_id)
         events = obs.events.select_energy(self.energy_band)
         events = events.select_offset(self.offset_band)
-        counts_map.fill(events=events)
-        counts_map.data = counts_map.data.value
-        self.maps["counts"] = counts_map
+        if (len(events) != 0):
+            counts_map.fill(events=events)
+            counts_map.data = counts_map.data.value
+            self.maps["counts"] = counts_map
+        else:
+            self.maps["counts"] = counts_map
+
+    def _has_counts(self, obs_id):
```

Comment:
@cdeil 
Just tell me if this is ok to fill with zero or if I have to skip it? after it will be ready to be merge I think!